clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -562,6 +563,7 @@ message SearchResponse {
         map<string, CompositeFieldValue> sortedFields = 4; // Sorted field name to value
         // Field name to highlighted text fragments
         map<string, Highlights> highlights = 5;
+        string explain = 6;
```

Comment:
We might need to add comments, and update *rst

Reply:
I'll add comments first. Explain should be part of nrtsearch request/response rst files. But these rst files do not exist yet. I'll create a ticket to add these rst files